license_sh/reporters/ConsoleReporter.py

```diff
@@ -10,6 +10,12 @@ def output(dependency_tree):
                 print(" ")
             else:
                 color = RED if node.license_problem else GREEN
+                normalized_info = (
+                    f', normalized as "{BOLD}{color}{node.license_normalized}{RESET}"'
+                    if node.license_normalized != node.license
+                    and node.license_normalized is not None
+                    else ""
+                )
                 print(
-                    f"{pre}{node.name} - {node.version} - {color}{node.license}{RESET}"
+                    f"{pre}{node.name} - {node.version} - {color}{node.license}{RESET}{normalized_info}"
```

Comment:
This should be added to json reporter as well

Reply:
@S4n60w3n it's a property of the node, so it's there automatically.